### PR TITLE
[Doc] Remake file external table

### PR DESCRIPTION
**Pull request by evelynzhaojie**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docs/data_source/file_external_table.md [hunk 1]
```diff
@@ -143,9 +226,11 @@ PROPERTIES
 );
 ```
 
-Example 3: Create a file external table named `table_1` and use the IAM user-based credential method to access all the ORC files under the file path `s3://bucket-test/folder1` in AWS S3.
+Example 3: Create a file external table and use **IAM** **user** to access **all the ORC files** under the file path in AWS S3.
```
Comment (EsoragotoSpirit) [inline docs/data_source/file_external_table.md:229]: `**IAM user**`

File: docs/data_source/file_external_table.md [hunk 2]
```diff
@@ -1,82 +1,175 @@
...
-| path    | Yes          | The path of the data file.<ul><li>If the data file is stored in HDFS, the path format is `hdfs://<IP address of HDFS>:<port>/<path>`.</li><li>If the data file is stored in Amazon S3, the path format is `s3://<bucket name>/<folder>/`.</li></ul>Note the following rules when you enter the path:<ul><li>If the value of the `path` parameter ends with `'/'`, such as `hdfs://x.x.x.x/user/hive/warehouse/array2d_parq/data'/'`, StarRocks treats it as a path. When you execute a query, StarRocks traverses all data files under the path. It does not traverse data files by using recursion.</li><li>If the value of the `path` parameter does not end with `'/'`, such as `hdfs://x.x.x.x/user/hive/warehouse/array2d_parq/data`, StarRocks treats it as a single data file. When you execute a query, StarRocks only scans the data file.</li></ul> |
-| format  | Yes          | The format of the data file. Only Parquet and ORC are supported. |
+##### AWS S3-compatible storage
+
+If you need to access an AWS S3-compatible storage system, such as MinIO, configure `StorageCredentialParams` as follows to ensure a successful integration:
+
+```SQL
+"aws.s3.enable_ssl" = "<true | false>",
```
Comment (EsoragotoSpirit) [inline docs/data_source/file_external_table.md:127]: `{ true | false }`
下同

**Review by EsoragotoSpirit — CHANGES_REQUESTED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**